docs/UpdaterConfig.md

```diff
@@ -48,28 +48,44 @@ Alerts are text messages attached to GTFS objects, informing riders of disruptio
 <!-- real-time-alerts BEGIN -->
 <!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
 
-| Config Parameter          |    Type   | Summary                  |  Req./Opt. | Default Value | Since |
-|---------------------------|:---------:|--------------------------|:----------:|---------------|:-----:|
-| type = "REAL_TIME_ALERTS" |   `enum`  | The type of the updater. | *Required* |               |   na  |
-| earlyStartSec             | `integer` | TODO                     | *Optional* | `0`           |   na  |
-| feedId                    |  `string` | TODO                     | *Optional* |               |   na  |
-| frequencySec              | `integer` | TODO                     | *Optional* | `60`          |   na  |
-| fuzzyTripMatching         | `boolean` | TODO                     | *Optional* | `false`       |   na  |
-| url                       |  `string` | TODO                     | *Required* |               |   na  |
+| Config Parameter          |       Type      | Summary                                                                      |  Req./Opt. | Default Value | Since |
+|---------------------------|:---------------:|------------------------------------------------------------------------------|:----------:|---------------|:-----:|
+| type = "REAL_TIME_ALERTS" |      `enum`     | The type of the updater.                                                     | *Required* |               |   na  |
```

Comment:
`type` is also 1.5 - I just checked, can you update it?

Reply:
I've added a few more version in 88b4a6dd689e7b2d1361b891f40c18a45010fad4